Discord.Addons.Interactive/InlineReaction/InlineReactionCallback.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Discord.Commands;
+using Discord.WebSocket;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Discord.Addons.Interactive
+{
+    public class InlineReactionCallback : IReactionCallback
+    {
+        public RunMode RunMode => RunMode.Sync;
+
+        public ICriterion<SocketReaction> Criterion { get; }
+
+        public TimeSpan? Timeout { get; }
+
+        public SocketCommandContext Context { get; }
+
+        public IUserMessage Message { get; private set; }
+
+        readonly InteractiveService interactive;
```

Comment:
Please fix the formatting of this variable.

Reply:
Fixed.